pkg/gooq/function.go

```diff
@@ -194,6 +194,191 @@ func RTrim(
 	return NewStringExpressionFunction("RTRIM", expressions...)
 }
 
+func Chr(
+	asciiCode NumericExpression,
+) StringExpression {
+	// TODO: add strict checking on asciiCode (i.e. make sure is not 0)
```

Comment:
from the docs, " The NULL (0) character is not allowed because text data types cannot store such bytes."
unsure if we should enforce it here or let postgresql complain.

Reply:
If we can't enforce it at compile time IMHO we should just let PostgreSQL complain.